var/spack/repos/builtin/packages/parallel/package.py

```diff
@@ -32,7 +32,12 @@ class Parallel(AutotoolsPackage):
     """
 
     homepage = "http://www.gnu.org/software/parallel/"
-    url      = "http://ftp.gnu.org/gnu/parallel/parallel-20160422.tar.bz2"
+    url      = "http://ftp.gnu.org/gnu/parallel/parallel-20170122.tar.bz2"
 
+    version('20170122', 'c9f0ec01463dc75dbbf292fd8be5f1eb')
     version('20160422', '24621f684130472694333709bd4454cb')
     version('20160322', '4e81e0d36902ab4c4e969ee6f35e6e57')
+
+    def check(self):
+        # The Makefile has a 'test' target, but it does not work
+        make('check')
```

Comment:
Looks like this got pulled in from my recent PR. Oh well, shouldn't cause a conflict.